### DISCUSSION TITLE
[question][antv/x6-vue-shape]有没有不依赖`vue-demi`的纯`Vue3`版？公司纯`vue3`环境项目禁止`vue-demi`

### DISCUSSION THREAD

**Discussion by CoderMonkie** [antvis/X6#4389] [Q&A]
有没有不依赖`vue-demi`的纯`Vue3`版？公司纯`vue3`环境项目禁止`vue-demi`

**Comment by zlecheng** [comment]
> 有没有不依赖`vue-demi`的纯`Vue3`版？公司纯`vue3`环境项目禁止`vue-demi`

你的项目依赖项里面又不会出现vue-demi
